packages/wrangler/src/dev.tsx

```diff
@@ -1413,10 +1413,14 @@ export function getResolvedBindings(
 
 	const maskedVars = maskVars(bindings, configParam);
 
-	printBindings({
-		...bindings,
-		vars: maskedVars,
-	});
+	printBindings(
+		{
+			...bindings,
+			vars: maskedVars,
+		},
+		undefined,
```

Comment:
Could think about having the second parameter as an object now that there are multiple. It would help keep it explicit and more readable, and would avoid having to specify `undefined`.

Reply:
https://github.com/cloudflare/workers-sdk/pull/6972/commits/5bc416c131992dd5494bdec5b9ded819dd979b98